src/app/browse-page/card-view/card-view.component.ts

```diff
@@ -63,20 +65,25 @@ export class CardViewComponent implements OnInit {
     if (itemIds.length) {
       this._assets.getAllThumbnails(itemIds)
         .then( allThumbnails => {
-          this.thumbnails = allThumbnails.filter((thumbnail) => {
-            return thumbnail.status === 'available'
-          })
-
-          this.thumbnails.forEach(thumbnail => {
-            if (thumbnail['media']) {
-              thumbnail.thumbnailImgUrl = thumbnail.media.thumbnailSizeOnePath
-            }
-          })
+          this.thumbnails = allThumbnails
+          // this.thumbnails = allThumbnails.filter((thumbnail) => {
+          //   return thumbnail.status === 'available'
```

Comment:
I'm pretty sure we should still be checking for status? As well as thumb path below?

Reply:
@bFraley